relayer/src/event/rpc.rs

```diff
@@ -99,13 +111,16 @@ pub fn build_event(mut object: RawObject) -> Result<IbcEvent, BoxError> {
         // - "register" and "send" for ICS27
         // However the attributes are all prefixed with "send_packet" therefore the overwrite here
         // TODO: This need to be sorted out
-        "transfer" | "register" | "send" => {
+        "transfer"
+        | "register"
+        | "send"
```

Comment:
could we update this for ICS27 as well?

Reply:
Not sure if my understanding is correct. 09ad7b8237c16d38cfa76b6079291084cc0c039c Please let me know if otherwise!

Reply:
Looks like [b912347](https://github.com/informalsystems/ibc-rs/pull/948/commits/b9123478151aed717b0e7f21d27fd2a592d7eaaf) resolved this